extensions/wikia/PhalanxII/classes/PhalanxPager.class.php

```diff
@@ -75,83 +80,82 @@ function getIndexField() {
 	}
 
 	function formatRow( $row ) {
 		// hide e-mail filters
-		if ( ( $row->p_type & Phalanx::TYPE_EMAIL ) && !$this->app->wg->User->isAllowed( 'phalanxemailblock' ) ) {
+		if ( ( $row->p_type & Phalanx::TYPE_EMAIL ) && !$this->getUser()->isAllowed( 'phalanxemailblock' ) ) {
 			return '';
 		}
 
-		if (isset($row->p_author_id)) {
-			$author = User::newFromId( $row->p_author_id );
+		// SUS-270: Get author from correct field
+		if ( isset( $row->p_authorId ) ) {
+			$author = User::newFromId( $row->p_authorId );
 			$authorName = $author->getName();
-		}
-		else {
+		} else {
 			$authorName = '';
 		}
 
 		$statsUrl = sprintf( "%s/%s", $this->phalanxStatsPage->getLocalUrl(), $row->p_id );
 
-		$html  = Html::openElement( 'li', array( 'id' => 'phalanx-block-' . $row->p_id ) );
-		$html .= Html::element( 'b', array('class' => 'blockContent'), $row->p_text );
+		$html = Html::openElement( 'li', [ 'id' => 'phalanx-block-' . $row->p_id ] );
+		$html .= Html::element( 'b', [ 'class' => 'blockContent' ], $row->p_text );
 		$html .= sprintf( " (%s%s%s) ",
-			( !empty($row->p_regex) ? 'regex' : 'plain' ),
-			( !empty($row->p_case)  ? ',case' : '' ),
-			( !empty($row->p_exact) ? ',exact': '' )
+			( !empty( $row->p_regex ) ? 'regex' : 'plain' ),
+			( !empty( $row->p_case ) ? ',case' : '' ),
+			( !empty( $row->p_exact ) ? ',exact' : '' )
 		);
 
 		/* control links */
 		$html .= sprintf( " &bull; %s &bull; %s",
-			Html::element( 'a', array(
+			Html::element( 'a', [
 				'class' => 'modify',
-				'href' => $this->phalanxPage->getLocalUrl( array( 'id' => $row->p_id ) )
-			), wfMsg('phalanx-link-modify') ),
-			Html::element( 'a', array(
+				'href' => $this->phalanxPage->getLocalUrl( [ 'id' => $row->p_id ] ),
+			], $this->msg( 'phalanx-link-modify' )->text() ),
+			Html::element( 'a', [
 				'class' => 'stats',
-				'href' => $statsUrl
-			), wfMsg('phalanx-link-stats') )
+				'href' => $statsUrl,
+			], $this->msg( 'phalanx-link-stats' )->text() )
 		);
 
 		/* remove block button - handled via AJAX */
-		$html .= Html::element( 'button', array(
+		$html .= Html::element( 'button', [
 			'class' => 'unblock',
 			'data-id' => $row->p_id,
-		), wfMsg('phalanx-link-unblock') );
+		], $this->msg( 'phalanx-link-unblock' )->text() );
 
-		$html .= Html::element('br');
+		$html .= Html::element( 'br' );
 
 		/* types */
-		$html .= wfMsg('phalanx-display-row-blocks', implode( ', ', Phalanx::getTypeNames( $row->p_type ) ) );
+		$html .= $this->msg( 'phalanx-display-row-blocks', implode( ', ', Phalanx::getTypeNames( $row->p_type ) ) )->escaped();
 
 		/* created */
-		if (isset($row->p_timestamp)) {
-			$html .= sprintf( " &bull; %s ",
-				wfMsgExt( 'phalanx-display-row-created', array('parseinline'),
-					$authorName,
-					$this->app->wg->Lang->timeanddate( $row->p_timestamp )
-				)
-			);
+		if ( isset( $row->p_timestamp ) ) {
+			$html .= ' &bull; ';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal " &bull; " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)